src/app-middleware/configureCors.js

```diff
@@ -4,6 +4,9 @@ const cors = require('cors');
 const config = require('../config');
 
 function parseValues(originalValues) {
+  if (originalValues === undefined) {
```

Comment:
This line is because we had to remove the default values for CORS. So we need to specify * to allow everything or else not load the middleware entirely when no CORS settings are specified.

I'm not sure what you're asking for regarding the secret. I added a new environment variable with that name but it's not being used anywhere.

Reply:
We just need to support that scenario; we don't need to actually add TOP_SECRET (though you'll need to to test that it works). As of now, it doesn't work:

> mocha --opts tests/unit.opts

/Users/bwortman/Documents/Projects/Fusion/node-express-api-skeleton/node_modules/12factor-config/index.js:49
                throw new Error("Required value not provided: " + name);
                ^

Error: Required value not provided: topSecret

Reply:
@BWortman Do you have the required values in your .env file? It's working on my machine